clang/docs/checkedc/Update-to-latest-LLVM-sources.md

```diff
@@ -23,13 +23,20 @@ The third step is to merge your changes back into your baseline and master branc
 First create remote to the mirrored GitHub repo that contains the updated sources
 for LLVM/Clang. Go to your LLVM/Clang repo and do:
 
-    git remote add mirror https://github.com/llvm-mirror/llvm-project
+    git remote add mirror https://github.com/llvm/llvm-project
 
 Then branch your baseline branch and merge changes into it:
 
     git checkout baseline
     git checkout -b updated-baseline
     git pull mirror master
+    git reset --hard <llvm-commit>
+
+`<llvm-commit>` is the commit on https://github.com/llvm/llvm-project that contains the desired version of the LLVM/Clang sources. For example, to update to LLVM/Clang version 11.0:
+
+    git reset --hard 176249bd6732a8044d457092ed932768724a6f06
```

Comment:
You can just use the tag name here, i.e., `git reset --hard llvmorg-11.0.0`, rather than manually copying and pasting the commit ID.